core/finality-grandpa/primitives/src/lib.rs

```diff
@@ -93,22 +93,44 @@ pub enum ConsensusLog<N: Codec> {
 	/// Note that the authority with given index is disabled until the next change.
 	#[codec(index = "3")]
 	OnDisabled(AuthorityIndex),
+	/// A signal to pause the current authority set after the given delay.
+	/// After finalizing the block at _delay_ the authorities should stop voting.
+	Pause(N),
```

Comment:
Why is this needed as a digest item? Surely digestitems are just for the benefit of light clients which don't vote anyway. An event is sufficient, no?